js/stores/appStore.js

```diff
@@ -563,6 +565,9 @@ const handleAppAction = (action) => {
         if (device.name) {
           appState = appState.setIn(['sync', 'devices', deviceId, 'name'], device.name)
         }
+        if (!hasMainDevice && getSetting(settings.SYNC_DEVICE_NAME) === device.name) {
```

Comment:
i think this line just sets the mainDevice to be true for any device with the same name as the current device, since `SYNC_DEVICE_NAME` is the name of the current device. is that the intended behavior?

Reply:
(as long as the current device doesn't know of any existing main devices)